lib/base-compiler.js

```diff
@@ -153,6 +153,7 @@ class BaseCompiler {
         let args = ["-d", outputFilename, "-l", "--insn-width=16"];
         if (demangle) args = args.concat("-C");
         if (intelAsm) args = args.concat(["-M", "intel"]);
+        if (this.lang.name === 'CUDA') args = [outputFilename, "-c", "-g", "-hex"]; // For nvdisasm
```

Comment:
I would prefer this to be part of a specialized objdjump function within class NvccCompiler

Reply:
Done.